app/javascript/components/teams-dropdown.vue

```diff
@@ -33,15 +33,19 @@ export default {
     return {
       dropdownTitle,
       noTeamsMessage,
+      monthSeparationDropdown: 3,
     };
   },
-  mounted() {
-    if (!this.adminMode) {
-      const selectedTeam = this.teams[this.defaultTeamIndex];
-      if (selectedTeam) {
-        this.onTeamSelected(selectedTeam);
+
+  watch: {
+    defaultTeamIndex(newValue) {
```

Comment:
ojo que el watch no se llama con el valor inicial se supone, para que le eches un ojo si eso puede traer problemas. [Acá](https://forum.vuejs.org/t/watchers-not-triggered-on-initialization/12475/16) hablan de un `immediate` que podría ayudar si se necesita

Reply:
como nunca se necesita para el primer render, solo para el segunda en adelante, en este caso estaría bien ;)